roles/keystone/templates/etc/keystone/policy.json

```diff
@@ -2,9 +2,11 @@
 
 {
 {% if policy_for_horizon|default('False')|bool %}
-    "admin_required": "role:admin or is_admin:1 or role:cloud_admin or role:project_admin or rule:owner",
+    "admin_required_limit": "role:admin or is_admin:1 or role:cloud_admin or role:project_admin or rule:owner",
```

Comment:
I still do not see why this change to the policy file is required. cloud_admin has all the the Keystone CRUD privileges.